src/app/public/modules/action-hub/action-hub.component.ts

```diff
@@ -17,30 +18,78 @@ let data: SkyActionHubData;
 export class SkyActionHubComponent {
   /**
    * Passes a SkyActionHubData object to build the action hub. The page loads until
-   * `title` has a value.
+   * `[data]` has a value.
    *
    * @param value
    */
   @Input()
   public set data(value: typeof data) {
```

Comment:
So our `docs-tools` seems to be struggling with this setter and lists this as `any`. Why are we specifying the `data` outside of the class instead of just doing

```
public set data(value: SkyActionHubData | undefined) {
```